camel/storages/graph_storages/nebula_graph.py

```diff
@@ -224,9 +232,15 @@ def ensure_tag_exists(self, tag_name: str) -> None:
         Raises:
             Exception: If the tag creation fails after retries, an exception
                 is raised with the error message.
+        
+        Add a `created_at` property to the tag with a default
+        value of the current timestamp
         """
 
-        create_tag_stmt = f'CREATE TAG IF NOT EXISTS {tag_name}()'
+        create_tag_stmt = f"""
+        CREATE TAG IF NOT EXISTS {tag_name}
+        (created_at DATETIME DEFAULT now())'
```

Comment:
could we add one more argument to allow user set the time? for some cases user may not want to add the now() time